### PR TITLE
zh： notice

### PR DESCRIPTION
**Pull request by loverto**
add

Signed-off-by: ylf <1045438139@qq.com>

### CODE REVIEW

**Comment by fleeto** [comment]
`[trans-bot:pushed]`

Thank you @loverto, the [related task issue](https://github.com/servicemesher/istio-official-translation/issues/843) had been updated.

File: content_zh/docs/setup/kubernetes/upgrade/notice/index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: 1.1 升级通知
+description: 在升级到 Istio 1.1 之前，运营商必须了解的重要更改。
```
Comment (rootsongjc) [inline content_zh/docs/setup/kubernetes/upgrade/notice/index.md]: 运营商？

File: content_zh/docs/setup/kubernetes/upgrade/notice/index.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+weight: 5
+
+---
+
+此页面描述了从 Istio 1.0 升级到 1.1 时需要注意的事项。在这里，我们详细介绍了我们向后不兼容性的情况。我们还提到了保留向后兼容性但引入了新行为的情况，这对于熟悉 Istio 1.0 的使用和操作的人来说是令人惊讶的。
+
+有关 Istio 1.1 引入的新功能的概述，请参阅 [1.1 发行说明](/about/notes/1.1/)。
```
Comment (rootsongjc) [inline content_zh/docs/setup/kubernetes/upgrade/notice/index.md]: 请链接到中文页面。

File: content_zh/docs/setup/kubernetes/upgrade/notice/index.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+## 安装
+
+- 我们增加了控制平面和 Envoy Sidecar 所需的 CPU 和内存。在更新之前，确保群集有足够的资源。
+- Istio 的 CRD 已被放入他们自己的 Helm chart `istio-init` 中。这可以防止丢失自定义资源数据，促进升级过程，使 Istio 能够基于 Helm 的安装形式也可以升级。 [升级文档](/docs/setup/kubernetes/upgrade/steps/) 提供了从 Istio 1.0.6 升级到 Istio 1.1 的正确过程。升级时请仔细遵循这些说明。如果需要 `certmanager`，在使用 `template` 或 `tiller` 安装模式安装`istio-init` 和 Istio chart 时，请使用 `--set certmanager=true` 标志。
+- 用于[多集群 VPN](/docs/setup/kubernetes/install/multicluster/vpn/) 的 1.0 `istio-remote` chart 和 [多集群水平分割](/docs/examples/multicluster/split-horizon-eds/) 远程集群安装已合并到 Istio chart 中。要生成等效的` istio-remote` chart，请使用 `--set global.istioRemote=true` 标志。
```
Comment (rootsongjc) [inline content_zh/docs/setup/kubernetes/upgrade/notice/index.md]: 文中都不会用中文页面链接地址吗？

Comment (loverto) [reply content_zh/docs/setup/kubernetes/upgrade/notice/index.md]: 这几个是调整目录的，我怕有的没有中文目录

**Comment by fleeto** [comment]
`[trans-bot:merged]`

Thank you @loverto, the [related task issue](https://github.com/servicemesher/istio-official-translation/issues/843) had been updated.

**Pull request merged**